deploy/LOCAL_INSTALL_HELM.md

```diff
@@ -0,0 +1,109 @@
+```yaml
+  seqr:
+    environment:
+      - SOCIAL_AUTH_AZUREAD_V2_OAUTH2_CLIENT_ID=your-client-id
+      - SOCIAL_AUTH_AZUREAD_V2_OAUTH2_SECRET=your-client-secret
+      - SOCIAL_AUTH_AZUREAD_V2_OAUTH2_TENANT=your-tenant-id 
+```
+Note that user accounts must be directly associated with the Azure tenant in order to access seqr. Anyone with access
+to the tenant will automatically have access to seqr, although they will only be able to view those projects that they 
+have been added to.
+
+## Enabling Clingen Allele Registration
+- Turning on this feature will register your variants within the Clingen Allele Registry during VCF ingestion. The [Registry](https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/landing) provides and maintains unique variant identifiers; enabling this feature will toggle several small features within *seqr*, including a link on the "Saved Variants" page to the Clingen Variant Curation Interface.
+- You will first need to [register](https://reg.clinicalgenome.org/cg-prof/new) and receive a login and password.
+- Create a kubernetes secret called `pipeline-secrets` with your login and password embedded:
+```bash
+kubectl create secret generic pipeline-secrets \
+  --from-literal=clingen_allele_registry_login='my-login'
+  --from-literal=clingen_allele_registry_password='my-password'
+```
+- Add secret references in your [values overrides](https://github.com/broadinstitute/seqr-helm?tab=readme-ov-file#valuesenvironment-overrides):
+```yaml
+pipeline-runner:
+  additionalSecrets:
+    - name: CLINGEN_ALLELE_REGISTRY_LOGIN
+      valueFrom:
+        secretKeyRef:
+          name: pipeline-secrets
+          key: clingen_allele_registry_login
+    - name: CLINGEN_ALLELE_REGISTRY_PASSWORD
+      valueFrom:
+        secretKeyRef:
+          name: pipeline-secrets
+          key: clingen_allele_registry_password
+```
+
+## Using the Load Data page to load VCF Callsets
+- Copy your vcf into the loading datasets directory on the node running your kubernetes cluster (`/var/seqr/seqr-loading-temp/`).  You should see your vcf present when listing files:
+```
+ls -h /var/seqr/seqr-loading-temp/
+loading_pipeline_queue  test.vcf.gz
+```
+- In the top header of *seqr*, click on the **Data Management** button.
+- In the subheader, click on **Load Data**.
+- Type the name of the callset path into the **Callset File Path** text box (without the directory prefix), and select the appropriate Sample Type (WES/WGS) and Genome Version (GRCh37/GRCh38) for your project.  The pipeline includes a sequence of validation steps to insure the validity of your VCF, but these may be skipped by enabling the **Skip Callset Validation**option.  We strongly recommend leaving validation enabled to ensure the quality of your analysis.
```

Comment:
Totally out of scope for documentation, but I made a future ticket for local installs to have a dropdown for VCFs instead of a free text box: https://github.com/broadinstitute/seqr-private/issues/1538